app/modules/search/controllers/searchControllers.js

```diff
@@ -126,22 +303,34 @@
                     }
                 }
             }
-            ;
-            //Perform search through client and get a search Promise
-            searchclient.search({
+            var request = {
                 index: API_CONF.ELASTIC_INDEX_NAME,
                 type: $scope.searchItemType,
                 body: {
                     size: $scope.itemsperPage,
                     from: itemOffset,
                     sort: sort,
-                    query: query
+                    query: query,
+                    aggs: requestAggs
                 }
-            }).then(function (resp) {
+            };
+            var filters = normalizeAggregationFilter();
+            if (!angular.equals({}, filters)) {
+                request.body.post_filter = filters;
+                //request.body.query = {
```

Comment:
I think i broke github :P The final result of this commit can be seen here: https://github.com/cbotsikas/policycompass-frontend/blob/issue138/app/modules/search/controllers/searchControllers.js#L317 and it's fine. The deleted lines 131 and 140 are now on a single line at 328.